src/components/editMeasure/editor/StatusHandler.tsx

```diff
@@ -18,6 +18,9 @@ const generateMadieAlertWithContent = (
   const warnings = warningAnnotation?.map((el) => {
     return <li>{transformAnnotation(el)}</li>;
   });
+  if (type === "success" && errorAnnotation && errorAnnotation.length > 0) {
+    type = "warning";
+  }
```

Comment:
I'm lil bit confused about this. why do we need to change this to warning?

Reply:
@adongare So this is when the cql is saved successfully, but there is error like no context. I guess we don't have to change it to warning, but it seems odd to me when there is error with the cql.

Reply:
yeah but then not having context makes whole cql invalid and if you go to population criteria or test case we tell user that there is an error with cql. therefore I think this should remain an error type and not warning

Reply:
okay, I checked lil bit more and its not consistent. I see error sometimes and success sometimes. maybe we can create new ticket and talk to POs. please ignore my previous messages.